mail-client/alpine/files/2.21.1-nopam.patch

```diff
@@ -0,0 +1,11 @@
+--- ./configure.ac	2019-08-13 21:08:23.353463222 +0200
++++ ./configure.ac	2019-08-13 21:08:36.611462488 +0200
+@@ -1108,7 +1108,7 @@
+       alpine_PAM="yes"
+     else
+       systype="LNX"
+-      AC_CHECK_LIB(pam, pam_start,
```

Comment:
It's probably quite not this simple :) did you test it gets disabled, and linked against with this if pam exists? 
https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Automagic_dependencies might be helpful.

Reply:
Thanks for the hint, you were right.

I fixed and tested it, it's not linked against PAM anymore if the USE flag is disabled.